co-simulation tool/docker/Dockerfile

```diff
@@ -0,0 +1,71 @@
+FROM ubuntu:18.04
+
+MAINTAINER ODU Team
+LABEL Description="Dockerised Simulation of Carla_Sumo_Mosaic"
+
+ENV SUMO_HOME /usr/share/sumo
+ENV SUMO_USER carla_sumo_mosaic
+
+RUN apt-get update && \
+    apt-get install -y openjdk-8-jdk && \
+    apt-get install -y ant && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/ && \
+    rm -rf /var/cache/oracle-jdk8-installer;
+
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
+RUN export JAVA_HOME
+
+RUN apt-get -y update
+
+RUN apt-get install x11-xserver-utils dconf-editor dbus-x11 -y
+
+RUN apt-get update && apt-get upgrade -y && apt-get clean
+
+# Python package management and basic dependencies
+RUN apt-get install -y curl python3.7 python3.7-dev python3.7-distutils
+
+# Register the version in alternatives
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.7 1
+
+# Set python 3 as the default python
+RUN update-alternatives --set python /usr/bin/python3.7
+
+RUN curl -s https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
+    python get-pip.py --force-reinstall && \
+    rm get-pip.py
+
+RUN pip install lxml
```

Comment:
It would be better to change pip install lxml to pip3 install lxml

Reply:
pip install lxml has been changed to pip3.7 install lxml in the dockerfile.